### DISCUSSION TITLE
Change global default format for table visualizations (numeric and date values)

### DISCUSSION THREAD

**Discussion by aesanchez** [getredash/redash#7243] [Q&A]
Hi Folks! 👋

Currently, every time I run a new query, I need to manually change the formatting for numeric values and date values in the visualization settings. This can be quite repetitive and time-consuming, especially when working with multiple queries or dashboards.

It would be amazing if there were a way to configure global default formats for table visualizations. For example:

- Setting a default number format (e.g., from 2 to X decimal places).
- Setting a default date format (e.g., from the default DD/MM/YY HH:mm to YYYY/MM/DD HH:mm.ss.SSS).

Ensuring that these defaults automatically apply to all new visualizations, reducing the need for manual adjustments.

Question:

1. Is this feature already available or in the pipeline?
2. If not, would it be possible to implement it as a global configuration option?

**Comment by aesanchez** [comment]
Found a question related to this, but without an answer > https://github.com/getredash/redash/discussions/7139

**Comment by mrolando-tn** [comment]
+1, would be a great addition :eyes:

**Comment by jgreenthal** [comment]
For date and time, Settings > General has some choices:
<img width="1076" alt="image" src="https://github.com/user-attachments/assets/c29b1e2c-8e5b-46cd-b7a9-12b6907596c9">
This is on v10.1.0.

My question #7139 (I filed it from a different account) was basically about applying the same concept to number formats.
